dotfiles/main.tf

```diff
@@ -14,7 +14,23 @@ variable "agent_id" {
   description = "The ID of a Coder agent."
 }
 
+variable "dotfiles_uri" {
+  type        = string
+  description = "The URL to a dotfiles repository. (optional)"
+
+  default = null
+}
+
```

Comment:
Why the variable?

Reply:
see https://github.com/coder/modules/pull/133#discussion_r1470937741